File: petgo_app/lib/features/content/domain/content_type.dart
/// 内容类型（与后端 `ContentType` 对齐）。「全部」是浏览态语义，不在此枚举（发布须落具体类型）。
enum ContentType {
  daily('DAILY'),
  growthMoment('GROWTH_MOMENT'),
  knowledge('KNOWLEDGE');

  const ContentType(this.wire);

  /// 后端枚举线格式（UPPER_SNAKE）。
  final String wire;
}

File: petgo-backend/src/main/java/com/tailtopia/content/domain/ContentType.java
package com.tailtopia.content.domain;

/**
 * 内容类型（FR-12，落库 varchar UPPER_SNAKE）。
 *
 * <ul>
 *   <li>{@link #DAILY}：日常分享。</li>
 *   <li>{@link #GROWTH_MOMENT}：成长日历快乐时刻——需绑定宠物档案，进档案时间线（2.4）。</li>
 *   <li>{@link #KNOWLEDGE}：专业科普。</li>
 * </ul>
 * 「全部」是浏览态语义，非可发布类型——发布时须落具体类型。
 */
public enum ContentType {
    DAILY,
    GROWTH_MOMENT,
    KNOWLEDGE
}
